bot/src/util.rs

```diff
@@ -0,0 +1,59 @@
+use chrono::{DateTime, Utc};
+use poise::serenity_prelude::{Color, Context, Message, Result, User};
+
+pub enum Penalty {
+    Timeout(DateTime<Utc>),
+    Ban(Option<DateTime<Utc>>),
+}
+
+pub async fn send_sanction_notification<S>(
+    ctx: &Context,
+    user: &User,
+    reason: S,
+    penalty: Penalty,
+) -> Result<Message>
+where
+    S: ToString,
+{
+    user.direct_message(ctx, |f| {
+        f.add_embed(|e| {
+            e.title("You've been sanctioned");
+            e.color(Color::RED);
+            e.description(format!(
+                "Hey {username},\n
+                I'm sorry to tell you, but you've been sanctioned for {reason}.
+                Your penalty is {penalty}
```

Comment:
Keep in mind that the spaces will also be in the final message afaik

Reply:
it is not ![image](https://user-images.githubusercontent.com/49766357/188751024-403cfd44-48a8-4997-ac94-04ccc89082d5.png)